src/tidy.zig

```diff
@@ -69,6 +81,117 @@ fn tidy_long_line(file: SourceFile) !void {
     }
 }
 
+/// `tigerbeetle/cli.zig`'s `parse_args` holds the function line coiunt record as of July 26, 2024
+/// with 366 lines. As we trim our functions, make sure to update this constant; tidy will
+/// error if you do not.
+const maximum_allowable_function_line_count = 366;
```

Comment:
>Add units or qualifiers to variable names, and put the units or qualifiers last, sorted by descending significance, so that the variable starts with the most significant word, and ends with the least significant word. For example, latency_ms_max rather than max_latency_ms. This will then line up nicely when latency_ms_min is added, as well as group all variables that relate to latency.

https://github.com/tigerbeetle/tigerbeetle/blob/main/docs/TIGER_STYLE.md#naming-things